dashboards/google-cloud-composer/metadata.yaml

```diff
@@ -0,0 +1,7 @@
+sample_dashboards:
+  -
+    category: Cloud Composer
+    id: cloud-composer-monitor
+    display_name: Cloud Composer Monitor
```

Comment:
It sounds like we have different names...

- display_name here is "Cloud Composer Monitor"
- title card in the dashboard is "Cloud Composer Monitor"
- display_name in the json dashboard definition is "cloud-composer-monitor"
- filenames are "cloud-composer-monitor.(json|png)"
- README.md table says "Cloud Composer Monitoring"

Sorry, I know I asked you in the earlier PR to use "Cloud Composer Monitor", but after looking at all the other GCP 1P dashboards, let's standardize to use "Cloud Composer Monitoring" and "cloud-composer-monitoring" everywhere -- in the filenames, in this metadata.yaml, in the README, in the dashboard JSON.

https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/tree/master/dashboards/google-cloud-functions
https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/tree/master/dashboards/google-cloud-run
https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/tree/master/dashboards/google-filestore
https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/tree/master/dashboards/google-microservices